scripts/run-for-subtyping.sh

```diff
@@ -3,33 +3,28 @@
 set -e
 set -o pipefail
 
-printf "Start generating pre-release files...\n\n"
+printf "Start molecular subtyping...\n\n"
 
-# Set locations for s3 bucket that contains subtyping commit files
+# Set locations for s3 bucket that contains release file
 URL="s3://d3b-openaccess-us-east-1-prd-pbta/open-targets"
-COMMIT="analysis-files-pre-release-commit"
+RELEASE="v12"
 
 # Set the working directory to the directory of this file
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
-# If RUN_LOCAL is used, the time-intensive steps are skipped because they cannot
-# be run on a local computer -- the idea is that setting RUN_LOCAL=1 will allow for
-# local testing running/testing of all other steps
-RUN_LOCAL=${RUN_LOCAL:-0}
-
 # Get base directory of project
 cd ..
 BASEDIR="$(pwd)"
 cd -
   
 analyses_dir="$BASEDIR/analyses"
-data_dir="$BASEDIR/data"
+data_dir="$BASEDIR/data/$RELEASE"
 scratch_dir="$BASEDIR/scratch"
 
-# Compile all the files that need to be examined for changes in one place
+# Compile all the files that need to be included in the release in one place
 # in the scratch directory
-commit_dir="${scratch_dir}/analysis-files-pre-release-commit"
-mkdir -p ${commit_dir}
+release_dir="${scratch_dir}/analysis-files-pre-release"
+mkdir -p ${release_dir}
```

Comment:
I am not sure we need to make this change here - we _do_ want all subtyping results in their respective directories.

Reply:
@ewafula can you check this

Reply:
@jharenza, look down at the scripts. The subtyping results are in their respective directories. In addition, only `histologies.tsv` and `independent sample list` are copied to the scratch temporary directory where other release files from `generate-analysis-files.sh` were previously copied, and `md5sum.txt` for release files updated in case someone needs to upload them to s3.  My assumption here is `generate-analysis-files.sh` + `run-for-subtypng.sh`  need to be run consecutively to generate release files. In the process, a collection of files only required for release is compiled in a folder in `scratch/`, which can be examined and then uploaded to s3. Nothing is being taken away from the module results folders.  
Maybe this is not what you want. We easily exclude the creation and copying of release files in a temporary folder in `scratch/` for both scripts that generate release files.

Reply:
no, this is perfect, thanks for clarifying